Gaijinizer patch/Enemies.txt

```diff
@@ -1482,13 +1453,11 @@ Strong against All#Triketos
 自身が召喚するバックダンサー達とダンス勝負で戦わせ、
 負けてしまった人間をショーの目玉として逆レイプする。
 > CONTEXT: Enemies/553/BookDescription/4
-M.O.W. Weapon Number E-002 "Idol".
-A girl who presides at "Human Reverse Raping SHOW" at Demon
-Realm TV.
-Fight against the backup dancers you summon in a dance
-match,
-We reverse rape as a highlight of the show with the people
-who lost.#MTLed
+M.O.W. Weapon E-002, "The Idol". A girl who hosts
+the "Human Reverse Raping SHOW" on Demon Realm TV.
+She makes humans compete in dance battles with
+her backup dancers, those who lose end up reverse
+raped as part of the show's climax.#Ron
 > END STRING
```

Comment:
Minor flow:
>_**M.O.W Weapon E-002: Idol.
 A girl who hosts the "Human Reverse Raping SHOW" on Demon Realm TV.**_